cornflow-server/cornflow/external_app/__init__.py

```diff
@@ -0,0 +1 @@
+from .model import *
```

Comment:
Why this change?

Reply:
It was so that the import_models()  method in cornflow-server/cornflow/shared/utils.py, that imports all of the models for the TablesEndpoint, could find the models of external_app. Otherwise they weren't getting imported

Reply:
Ok, but that part from external_app is getting moved, so we will have to see how to adapt it for the external_app later on. I will make a fast commit to delete ok?

Reply:
Okay